blog/views.py

```diff
@@ -0,0 +1,42 @@
+from django.contrib.auth.mixins import LoginRequiredMixin
+from django.http import HttpRequest, HttpResponse
+from django.shortcuts import render, redirect
+from django.core.paginator import Paginator, EmptyPage
+from django.urls import reverse_lazy, reverse
+from django.views import generic
+
+from blog.models import Post, Commentary
+
+
+def index(request: HttpRequest) -> HttpResponse:
```

Comment:
Rewrite all func views to class views

Reply:
I don't know how to rewrite comment create func to the class based. Can you give some advices?